cluster-autoscaler/utils/kubernetes/factory.go

```diff
@@ -30,7 +30,7 @@ import (
 // CreateEventRecorder creates an event recorder to send custom events to Kubernetes to be recorded for targeted Kubernetes objects
 func CreateEventRecorder(kubeClient clientset.Interface) kube_record.EventRecorder {
 	eventBroadcaster := kube_record.NewBroadcaster()
-	eventBroadcaster.StartLogging(glog.Infof)
+	eventBroadcaster.StartLogging(glog.V(2).Infof)
```

Comment:
Maybe V(3)? Logging events gets spammy on larger clusters (hundreds of PodTriggeredScaleUp or NotTriggeredScaleUp events).

Reply:
Changed to V(4) - the same as other debug-like logs.